examples/test_util/EclOutput.cpp

```diff
@@ -0,0 +1,400 @@
+/*
+   Copyright 2019 Statoil ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+   */
+
+#include "EclOutput.hpp"
+#include "EclUtil.hpp"
+#include <opm/common/ErrorMacros.hpp>
+
+#include <algorithm>
+#include <iterator>
+#include <iomanip>
+#include <stdexcept>
+#include <typeinfo>
+#include <sstream>
+#include <stdio.h>
+
+
+EclOutput::EclOutput(std::ofstream &inputFileH)
```

Comment:
The `EclFile` constructor takes a *filename* as argument and manages the stream internally - I would suggest doing the same thing here.

Reply:
done.